features/support/world.js

```diff
@@ -43,38 +43,33 @@ function ensurePipelineExists(config) {
         .then((response) => {
             this.jwt = response.body.token;
 
-            return request({
-                uri: `${this.instance}/${this.namespace}/pipelines`,
-                method: 'POST',
-                auth: {
-                    bearer: this.jwt
-                },
-                body: {
-                    checkoutUrl:
-                        `git@${this.scmHostname}:${this.testOrg}/${config.repoName}.git#master`
-                },
-                json: true
-            });
+            return this.createPipeline(config.repoName);
```

Comment:
Originally, pipelines are created in this function even before this PR. 
This function ensures that pipeline exists eventually at the end of this function.
So, if pipeline does not exist it will be created, and if does exist it will be deleted and re-created. This `delete and re-create` part is what I changed in this PR.